source/chemistry/bcl_chemistry_fragment_add_med_chem.cpp

```diff
@@ -240,6 +241,59 @@ namespace bcl
       // mutate label
       BCL_MessageStd( "AddMedChem!");
 
+      // this will cause issues so it's banned
+      if
+      (
+         ( m_TargetMoleculeLinkElementType.empty() && m_EnableDummyAtom ) ||
```

Comment:
Why not just have m_EnableDummyAtom=!m_TargetMoleculeLinkElementType.empty(); in the rest of the logic. If I give a list of linked element types, it seems I must want them, no?